chalk-integration/src/lowering/program_lowerer.rs

```diff
@@ -53,13 +53,13 @@ impl ProgramLowerer {
     pub fn extract_associated_types(
         &mut self,
         program: &Program,
-        raw_ids: &Vec<RawId>,
+        raw_ids: &[RawId],
```

Comment:
Backwards compatibility note: I don't think this fn is available outside the crate since `ProgramLowerer` isn't public, so this should be fine.